pkg/operator/certrotation/client_cert_rotation_controller.go

```diff
@@ -44,23 +70,21 @@ type CertRotationController struct {
 	RotatedSelfSignedCertKeySecret RotatedSelfSignedCertKeySecret
 
 	// Plumbing:
-	OperatorClient v1helpers.StaticPodOperatorClient
+	StatusReporter StatusReporter
 }
 
 func NewCertRotationController(
```

Comment:
No owner here?  Or optional With or something?

Reply:
Sorry, I don't follow the comment - could you please clarify?

Reply:
When I create a cert rotation order, can I wire up an ownerref for everything created by the operator during construction?

Reply:
I'm not sure if the implementation in this PR is exactly what you're looking for, but, yes - the constituent parts of the rotator allow for an OwnerRef to be passed in.